src/terminal/terminal/terminal-priv.h

```diff
@@ -325,23 +290,31 @@ struct guac_terminal {
     guac_terminal_display* display;
 
     /**
-     * Current terminal display state. All characters present on the screen
-     * are within this buffer. This has nothing to do with the display, which
-     * facilitates transfer of a set of changes to the remote display.
+     * The default, "normal" buffer containing all characters that should be
+     * displayed within the terminal emulator while not using the alternate
+     * buffer. Unless switched to the alternate buffer, all terminal operations
+     * will involve this buffer. The buffer that is relevant to terminal
+     * operations is determined by the current value of current_buffer.
      */
-    guac_terminal_buffer* buffer;
+    guac_terminal_buffer* normal_buffer;
 
     /**
-     * Alternate buffer.
+     * The non-default, "alternate" buffer containing all characters that should
+     * be displayed within the terminal emulator while not using the normal
+     * buffer. Unless switched to the alternate buffer, all terminal operations
```

Comment:
Copypasta? "Unless switched to the alternate buffer, all terminal operations will involve this buffer"/